backend/Server/DatabaseConnection.java

```diff
@@ -0,0 +1,161 @@
+package Server;
+
+import java.sql.*;
+import java.util.*;
+import java.lang.StringBuilder;
+
+public class DatabaseConnection implements AutoCloseable {
+    final static private String hostname = "scheduledsmiles.cdmwceky6go6.us-west-1.rds.amazonaws.com";
+    final static private int port = 3306; // 3306 is default
+    final static private String databaseName = "scheduledSmiles"; // case sensitive, will refuse connection if the name doesn't match.
+    final static private String username = "root";
+    final static private String password = "password";
+    static private String databaseUrl;

```

Comment:
Is databseUrl dynamic?

Reply:
It is not and should not be, but I can't set it as final since it's defined in the constructor.